example/ios/ReactNativeTHEOplayer/AppDelegate.m

```diff
@@ -30,6 +32,13 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
+  
+  NSString *receiverAppID = @"CC1AD845"; // default THEOplayer v3 receiver
```

Comment:
should not we parse the `receiverAppID` from the React PlayerConfiguration ?

Reply:
This is an initial default receiver we configure early as suggested bij [(react-native-google-cast)](https://react-native-google-cast.github.io/docs/getting-started/setup). We noticed that when not configured early (e.g. till we bridged the appId), the react-native-google-cast button does not show.
When we receive the bridged playerConfig, we indeed parse the correct receiverAppId and reconfigure the setup
(see [parsing](https://github.com/THEOplayer/react-native-theoplayer/blob/feature/casting/ios/THEOplayerRCTView.swift#:~:text=let%20castReceiverApplicationId%20%3D%20chromecastConfig%5B%22appID%22%5D%20as%3F%20String%20%7B ) end [re-config](https://github.com/THEOplayer/react-native-theoplayer/blob/feature/casting/ios/THEOplayerRCTView.swift#:~:text=let%20options%20%3D%20GCKCastOptions(discoveryCriteria%3A%20GCKDiscoveryCriteria(applicationID%3A%20castReceiverApplicationId)))